ops/terraform/services/common/aurora-cluster.tf

```diff
@@ -145,11 +145,6 @@ resource "aws_rds_cluster_endpoint" "readers" {
   cluster_identifier          = aws_rds_cluster.aurora_cluster.id
   cluster_endpoint_identifier = "bfd-${local.env}-ro"
   custom_endpoint_type        = "READER"
-
-  # EXCLUDED_MEMBERS assigns ALL but the last reader to the custom endpoint
-  excluded_members = [
-    element(local.reader_nodes, length(local.reader_nodes) - 1).id
-  ]
```

Comment:
I think we should keep this PR more focused and leave this adjustment for [BFD-2571](https://jira.cms.gov/browse/BFD-2571). This makes no functional difference–even with terraform applied, this custom reader still doesn't include the third node ☹️

Reply:
Yeah, I'm fine either way. I was debating removing it versus not removing, but keeping it makes no functional difference.

I'll revert this change.